lib/tasks/db.rake

```diff
@@ -0,0 +1,148 @@
+      FileUtils.touch Rails.root.join 'database-dumps', 'crimethinc_production_db_dump.sql'
+
+      # URL to download from
+      url = 'https://s3.amazonaws.com/thecloud.crimethinc.com/database-dumps/crimethinc_production_db_dump.sql'
+
+      puts '==> Downloading remote production DB dump from S3…'
+      open('database-dumps/crimethinc_production_db_dump.sql', 'wb') do |file|
+        file << open(url).read
+      end
+    end
+
+    desc 'Import pg dump into local development DB'
+    task import: :environment do
+      puts '==> Dropping local development DB…'
+      sh 'DISABLE_DATABASE_ENVIRONMENT_CHECK=1 rake db:drop'
+      puts
+
+      puts '==> Creating local test DB…'
+      sh 'rake db:create'
+      puts
+
+      puts '==> Migrating DB…'
+      sh 'rake db:migrate'
+      puts
+
+      puts '==> Populate DB from pg dump file…'
+      sh 'psql crimethinc_development < database-dumps/crimethinc_production_db_dump.sql'
+    end
+
+    desc 'Pull production DB'
+    task export: %i[pull scrub dump upload]
+
+    desc 'Pull production DB'
+    task pull: :environment do
+      puts '==> This requires that you have Heroku access to this production app'
+      puts
+
+      puts '==> Dropping local development DB…'
+      sh 'DISABLE_DATABASE_ENVIRONMENT_CHECK=1 rake db:drop'
+      puts
+
+      puts '==> Pulling remote production DB…'
+      sh 'heroku pg:pull DATABASE_URL crimethinc_development --app crimethinc'
+      puts
+
+      puts '==> Creating local test DB…'
+      sh 'rake db:create'
+    end
+
+    desc 'Scrub private production data from DB'
+    task scrub: :environment do
+      puts '==> Scrubbing DB…'
+      puts '==> Scrubbing Users…'
+
+      # delete all users except User ID #1, to reuse that ID
+      User.all.each do |user|
+        user.id == 1 ? next : user.destroy
+      end
+
+      # update the first user instead of creating a new one,
+      # so to not reveal how many production users there are
+      password = '1234567890' * 3
+      publisher_role = User::ROLES.index :publisher
+
+      User.find(1).update username:              :publisher,
+                          password:              password,
+                          password_confirmation: password,
+                          role:                  publisher_role
+
+      puts '==> Scrubbing drafts…'
+      # TODO: add #publication_status to these models: Definition Episode Podcast
+      [Article, Book, Issue, Journal, Logo, Page, Poster, Sticker, Video, Zine].each do |klass|
+        klass.draft.destroy_all
+      end
+    end
+
+    desc 'Dump local development DB'
+    task dump: :environment do
+      # ensure that the db dumps directory exists
+      FileUtils.mkdir_p Rails.root.join 'database-dumps'
+
+      puts '==> Dumping local development DB…'
+      # create a PG dump and save it to the db dumps directory
+      sh 'pg_dump --dbname=crimethinc_development --file=database-dumps/crimethinc_production_db_dump.sql'
```

Comment:
Because it's originally a dump of production data. I'm just pulling it to my laptop to do the data scrubbing before uploading it.

When we figure out some Heroku automation, it'll actually be a `pg_dump` from production.